### PR TITLE
pi3topbm, pbmtopi3: add pages

### PR DESCRIPTION
**Pull request by gutjuri**
- [x] The page(s) are in the correct platform directories: \`common\`, \`linux\`, \`osx\`, \`windows\`, \`sunos\`, \`android\`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

### CODE REVIEW

File: pages/common/pbmtopi3.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# pbmtopi3
+
+> Convert a PBM image into an Atari Degas PI3 image.
```
Comment (vitorhcl) [inline pages/common/pbmtopi3.md:3]:
```suggestion
> Convert a PBM image to an Atari Degas PI3 image.
```

File: pages/common/pbmtopi3.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+> See also: `pi3topbm`.
+> More information: <https://netpbm.sourceforge.net/doc/pbmtopi3.html>.
+
+- Convert a PBM image into an Atari Degas PI3 image:
```
Comment (vitorhcl) [inline pages/common/pbmtopi3.md:7]:
```suggestion
- Convert a PBM image to an Atari Degas PI3 image:
```

File: pages/common/pbmtopi3.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+`pbmtopi3 {{path/to/file.pbm}} > {{path/to/output.pi3}}`
```
Comment (vitorhcl) [inline pages/common/pbmtopi3.md:9]:
```suggestion
`pbmtopi3 {{path/to/image.pbm}} > {{path/to/atari_image.pi3}}`
```
Maybe we should use `output.pi3` for consistency with the other pages, but I prefer clarity over consistency in this specific case.

Not that the description is not clear, but this way one can directly look up the command example and have a good idea of what it does

File: pages/common/pi3topbm.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# pi3topbm
+
+> Convert an Atari Degas PI3 image into PBM image.
+> See also: `pbmtopi3`.
+> More information: <https://netpbm.sourceforge.net/doc/pi3topbm.html>.
+
+- Convert an Atari Degas PI3 image into PBM image:
+
+`pi1topbm {{path/to/file.pi3}} > {{path/to/output.pbm}}`
```
Comment (vitorhcl) [inline pages/common/pi3topbm.md:9]:
```suggestion
`pi1topbm {{path/to/atari_image.pi3}} > {{path/to/output_image.pbm}}`
```
See https://github.com/tldr-pages/tldr/pull/12527#discussion_r1530682854

**Review by vitorhcl — APPROVED** [review]
LGTM! I have some minor suggestions

File: pages/common/pbmtopi3.md (hunk 3 above)
Comment (gutjuri) [reply pages/common/pbmtopi3.md:9]: sure

**Review by kbdharun — APPROVED** [review]

**Pull request merged**